docs/introduction/install.rst

```diff
@@ -55,14 +55,14 @@ Installing on Linux OS:
   
      git clone git@github.com:HorizenOfficial/Sidechains-SDK.git
      
-  5. As IDE, please install and use IntelliJ IDEA Community Edition (link) In the IDE, please also install the Intellij Scala plugin: in the Settings->Plugins tab, select it from the marketplace: 
+  5. As IDE, please install and use IntelliJ IDEA Community Edition (link). In the IDE, please also install the Intellij Scala plugin: in the Settings->Plugins tab, select it from the marketplace: 
   
   .. image:: /images/intellij.png
    :alt: IntelliJ
   
-  6. In the IDE, you can now  go to File and Open the root directory of the project repository, “\Sidechains-SDK”. The pom.xml file - the Maven Project Object Model XML file that contains all the project configuration details - should be automatically imported by the IDE. Otherwise, you can just open it.
+  6. In the IDE, you can now  go to File and Open the root directory of the project repository, “\Sidechains-SDK”. The pom.xml file - the Maven Project's Object Model XML file that contains all the project configuration details, should be automatically imported by the IDE. Otherwise, you can just open it.
```

Comment:
@zennermac   

6. In the IDE, you can now  go to File and Open the root directory of the project repository, “\Sidechains-SDK”. The pom.xml file - the Maven Project's Object Model XML file that contains all the project configuration details should be automatically imported by the IDE. Otherwise, you can just open it.

@zennermac - this section doesn't make sense to me. Can you message me about this. "The pom.xml file - the Maven Project's Object Model XML file"